src/cmd_vel_mux.cpp

```diff
@@ -234,68 +274,170 @@ rcl_interfaces::msg::SetParametersResult CmdVelMux::parameterUpdate(
   rcl_interfaces::msg::SetParametersResult result;
   result.successful = true;
 
-  std::vector<std::string> names;
-  std::vector<std::string> topics;
-  std::vector<double> timeouts;
-  std::vector<int64_t> priorities;
-  std::vector<std::string> short_descs;
+  std::map<std::string, rclcpp::Parameter> subs_parameters;
```

Comment:
Hm, all of this code below is pretty complicated now.  I think I'd suggest that this should be refactored like the code in the constructor to be simpler.  It might make sense to make a common method that deals with both.